LondonBritishLibrary/orient/BLorient8815.xml

```diff
@@ -0,0 +1,274 @@
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.1.15">
+                                        <locus from="28v" to="29r"/>
+                                        <title ref="LIT1828Mahale#Simeon"/>
+                                    </msItem>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.2">
+                                    <locus from="94r" to="97v"/>
+                                    <title ref="LIT2362Songof" type="incomplete"/>
+                                    <note>The text is divided into five sections. Desinit ex abrupto (VIII, 2).</note>
+                                    <msItem xml:id="ms_i1.2.2.1">
+                                        <title ref="LIT2362Songof#FirstCanticle"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.2">
+                                        <title ref="LIT2362Songof'#SecondCanticle"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.3">
+                                        <title ref="LIT2362Songof#ThirdCanticle"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.4">
+                                        <title ref="LIT2362Songof#FourthCanticle"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.5">
+                                        <title ref="LIT2362Songof#FifthCanticle" type="incomplete"/>
+                                    </msItem>
+                                </msItem>
+                            </msItem>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">2+35</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>142</height>
+                                        <width>110</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="21"/>
+                            </layoutDesc>    
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Written in good hand.</desc>
```

Comment:
in a good hand?

Reply:
hand=handwriting, no article in English )